twindle-cli/src/index.js

```diff
@@ -120,6 +129,15 @@ function calculateFileName(cliObject, data) {
   return intelligentOutputFileName;
 }
 
+function calculateFileNameForGitHub(cliObject, data) {
+   let userName = data[0].common.user.username;
```

Comment:
In the Twitter part when we're doing the intelligent filename generation, a statement like this breaks down badly.

I don't know if this is the case for the github API, but I would request @Mira-Alf who knows of that intelligentFileName function as well the github API intricasies to look at it